platform/mds/mds-test-bundle/src/main/java/org/motechproject/mds/test/domain/Discr1Start.java

```diff
@@ -0,0 +1,47 @@
+package org.motechproject.mds.test.domain;
+
+import org.motechproject.mds.annotations.Entity;
+import org.motechproject.mds.annotations.Field;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+public class Discr1Start
```

Comment:
I'm already lost, can we rename this class or at least add a comment explaining the structure?